### PR TITLE
Docker Compose Addons Fix

### PR DESCRIPTION
**Pull request by Michael7371**
Changes the `docker-compose-addons.yml` file to reference the parent `docker-compose.yml` file to remove code duplication.

### CODE REVIEW

File: docker-compose-addons.yml [hunk 1]
```diff
@@ -223,9 +89,5 @@ services:
         max-size: '10m'
         max-file: '5'
 
-volumes:
-  pgdb:
-    driver: local
-
 networks:
```
Comment (dmccoystephenson) [inline docker-compose-addons.yml]: I'm running into the following error:

> imported compose file [/home/wyocvadmin/cvmanager/jpo-cvmanager/docker-compose.yml] defines conflicting network internal

I think the 'internal' network declaration needs to be removed from this file.

**Review by dmccoystephenson — CHANGES_REQUESTED** [review]
Ran into an error running the `docker-compose-addons.yml` file. Looks like a duplicate network declaration error.

File: docker-compose-addons.yml (hunk 1 above)
Comment (Michael7371) [reply docker-compose-addons.yml]: I removed the internal network for both files. I don't think it is necessary.

**Review by dmccoystephenson — APPROVED** [review]
Looks good to me!

**Review by drewjj — APPROVED** [review]
Looks good!

**Pull request merged**